/usr/bin/python3: can't open file '/mnt/c/dev/loot/VF3/export_satsuki_modular.py': [Errno 2] No such file or directory
